docs/guides/main.rst

```diff
@@ -139,6 +146,11 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
+For Windows user, this would look like::
+
+    directory: "D:\\My Music Files"
+    library: "D:\\My Music Files\\Musiclibrary.db"
```

Comment:
Right; you can use Windowsy paths too—I just wanted to point out that either works fine. And spaces work fine too; no need to handle them any differently.